pages/reference-service.html

```diff
@@ -4,16 +4,48 @@
 date: 2014-02-20 06:56:35.000000000 +08:00
 permalink: /odata-services/
 ---
+
+<style>
+.btn {
+color: #f58d42;
+background-color: white;
+}
+
+.btn:hover{
+color: #f58d42;
+background-color: white;
+}
+
+.btn bottom.active  {
+}
+
+.btn:focus  {
+color: #f58d42;
+}
+
+
+</style>
+
+<script>
+function activateTabsById(id) 
```

Comment:
Kindly look at https://www.w3.org/TR/wai-aria-practices-1.1/examples/tabs/tabs-1/tabs.html to retain the current activation. Tabs.js i believe is in the project already.

Reply:
@wandeg can guide you on this.

Reply:
Yeah let's reuse existing functionality where possible. Tabs.js already allows you to switch between tabs by clicking or key scrolling. Looks like the only thing required would be to add the css.